bot/resources/evergreen/py_topics.yaml

```diff
@@ -23,6 +23,19 @@
     - When you were first learning, what is a resource you wish you had?
     - What is something you know now, that you wish you knew when starting out?
     - What is something simple that you still error on today?
+    - What do you plan on eventually achieving with python?
+    - Is python your first programming language, if not, what is?
+    - What's your favourite aspect of python development? (Backend web dev/game dev/machine learning and ai ect)
+    - In what ways has python discord helped you with python?
+    - What part of python is currently dieing?
+    - Are you currently a student, or using python professionally?
```

Comment:
Mentioning something about hobby programmers would be nice here, as there are many of those.